assets/js/modules/analytics/components/dashboard/DashboardAllTrafficWidget/UserCountGraph.js

```diff
@@ -233,7 +233,7 @@ UserCountGraph.chartOptions = {
 		left: '1%',
 		height: 300,
 		top: 21,
-		width: '90%',
+		width: '89%',
```

Comment:
Not saying the change here is wrong at all, but I'm curious why this is like that in the first place? Why do we have `left: '1%'` here when elsewhere we don't use percentages for that? Also why is the total 90% but not 100%?

Was this potentially only due to a previous effort to solve truncations? If so, maybe we can try our new approach here as well.

Reply:
Further on the above, I've pushed the proposed change to `chartArea` in case anyone wants to try it out, hence this comment thread now appearing as "Outdated".